docs/nydusd.md

```diff
@@ -8,27 +8,41 @@ Get `nydusd` binary from [release](https://github.com/dragonflyoss/image-service
 
 ## Run Nydusd Daemon
 
-``` shell
+```shell
 # Prepare nydusd configuration
-cat /path/to/config-localfs.json
+sudo tee /etc/nydus/nydusd-config.localfs.json > /dev/null << EOF
 {
   "device": {
     "backend": {
       "type": "localfs",
       "config": {
-        "dir": "/path/to/blobs",
+        "dir": "/tmp/blobs/"
```

Comment:
how about using `/var/lib/nydus/blobs` here?

Reply:
Looks good. I fixed it.